packages/core/strapi/tests/endpoint.test.e2e.js

```diff
@@ -197,6 +221,30 @@ describe('Create Strapi API End to End', () => {
       expect(body.data.attributes.tags.data.length).toBe(3);
     });
 
+    test('Update article1 adding a non existent tag', async () => {
+      const { id, attributes } = data.articles[0];
+      const entry = { ...attributes };
+      entry.tags = [1000, 1001, 1002];
```

Comment:
What do you think of adding one existing relation here? So we can see that the count is correct and that it doesn't validate even when some are correct